symfony/lib/model/doctrine/orangehrmBuzzPlugin/BuzzNotificationMetadata.class.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * BuzzNotificationMetadata
+ * 
+ * This class has been auto-generated by the Doctrine ORM Framework
+ * 
+ *  
+ * @package    orangehrm
```

Comment:
I know this is generated stuff.. any chance to chnage this template?

Reply:
I'll look into that